controller/sync.go

```diff
@@ -56,6 +56,18 @@ func (m *appStateManager) getGVKParser(server string) (*managedfields.GvkParser,
 	}
 	return cluster.GetGVKParser(), nil
 }
+func (m *appStateManager) getResourceOperations(server string) (kube.ResourceOperations, func(), error) {
+	clusterCache, err := m.liveStateCache.GetClusterCache(server)
+	if err != nil {
+		return nil, nil, fmt.Errorf("error getting cluster cache: %w", err)
+	}
+
+	cluster, err := m.db.GetCluster(context.Background(), server)
+	if err != nil {
+		return nil, nil, fmt.Errorf("error getting cluster: %w", err)
+	}
+	return clusterCache.GetKubectl().ManageResources(cluster.RawRestConfig(), clusterCache.GetOpenAPISchema())
```

Comment:
Do we actually need to use `clusterCache.GetKubectl()` here? I think we already have `m.kubectl`, which seems like it should be sufficient. It would let us avoid exposing another public method on the cluster cache.

Reply:
Good catch! `m.kubectl` is indeed sufficient. Maybe outdated code from when I implemented the first draft.. 🤔

Reply:
removed the kubectl getter from the gitops-engine PR.